src/H5webApp.tsx

```diff
@@ -1,25 +1,7 @@
-import '@h5web/app/dist/style-lib.css';
-import '@h5web/app/dist/style.css';
```

Comment:
The CSS is not imported when imported from JS so I import it from the CSS :shrug: